Client/src/Remote/Endpoints/DimensionsEndpoint.tsx

```diff
@@ -0,0 +1,42 @@
+import SnackbarUtils from "../../Components/Utils/SnackbarUtils"
+import { get, post, put, _delete } from "../FluentRequest"
+
+import { IDimensionModel, IDimensionQueryModel, IUnitModel } from "../../../../Server/src/models/interfaces/IDimension"
+
+const dimensionRoute = '/api/v1/dimension'
+
+// /api/v1/dimensions - POST - dimension model
+export const callAddDimension = async (dimensionModel: IDimensionModel): Promise<any> => {
+    const result: IUnitModel[] = await post(dimensionRoute).withBody(dimensionModel).json()
+    // if (result == 'Success') {
```

Comment:
nice that you put snackbars, uncomment these lines I wanted to add them later too for all the other endpoints